README.md

```diff
@@ -41,7 +41,7 @@ To successfully install Kafka using this module you need to have Apache Zookeepe
 already running at localhost:2181. You can specify another Zookeeper host:port
 configuration using the config hash of the kafka:broker class.
 
-The default configuration installs Kafka 0.9.0.1 binaries with Scala 2.11
+The default configuration installs Kafka 0.11.0.1 binaries with Scala 2.11
```

Comment:
should be scala 2.12?

Reply:
The recommended version is still 2.11. See also https://issues.apache.org/jira/browse/KAFKA-6090.